packages/babel-plugin-emotion/src/utils/transform-expression-with-styles.js

```diff
@@ -91,10 +98,29 @@ export let transformExpressionWithStyles = ({
       ])
       let node = prodNode
       if (sourceMap) {
+        if (!state.emotionStringifiedCssId) {
+          const uid = state.file.scope.generateUidIdentifier(
+            '__EMOTION_STRINGIFIED_CSS_ERROR__'
+          )
+          state.emotionStringifiedCssId = uid
+          const cssObjectToString = t.functionDeclaration(
+            uid,
+            [],
+            t.blockStatement([
+              t.returnStatement(t.stringLiteral(CSS_OBJECT_STRINGIFIED_ERROR))
+            ])
+          )
+          cssObjectToString._compact = true
```

Comment:
What's this?

Reply:
A hint for babel generator, in this case it doesnt matter much - can remove it if you prefer to omit this. 

It basically tells generator to print the statement in a single line, so instead of
```js
function a() {
  return 42
}
```
it prints
```
function a() { return 42 }
```

Babel is using this flag when inserting helpers.